src/main/java/com/example/Restaurant_Management_System_REST_API/model/SingleRecord.java

```diff
@@ -0,0 +1,15 @@
+package com.example.Restaurant_Management_System_REST_API.model;
+
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@MappedSuperclass
+public abstract class SingleRecord {
```

Comment:
I don't like the name. It's very vague what it represents.

Reply:
it is represents a single record in restaurant menu and in inventory list.  I made it abstract because in both places I need such fields and I didn't want to duplicate code.

Reply:
Would you be able to find a better name for it? I think there might be better names for that case.

Reply:
Maybe you can propose something? I don't want to guess what you have in your mind and I don't want to play in guessing game like "or maybe this?  not? so maybe this? or maybe this? ". 
I just can tell you that `Record` cannot be -> IntelliJ takes it as a interior part of Java class.
What I could propose else than SingleRecord is `CatalogItem`.
if you have other concept for this name please share with me

Reply:
I just asked if you have anything in your mind cause it sounded like that :)
Ok, so I rename it to CatalogItem for now and later we will see!